openfl-workspace/keras_nlp/src/nlp_dataloader.py

```diff
@@ -47,11 +51,11 @@ def __init__(self, collaborator_count, split_ratio,
         self.X_valid = [valid[0], valid[1]]
         self.y_valid = valid[2]
 
-    def get_feature_shape(self):
+    def get_feature_shape(self) -> Tuple[int, ...]:
         """Get the shape of an example feature array."""
         return self.X_train[0].shape
 
-    def get_train_loader(self, batch_size=None):
+    def get_train_loader(self, batch_size: Optional[int] = None) -> Iterator[List[np.ndarray]]:
```

Comment:
`batch_size` couldn't be `None`

Reply:
I don't think that we should change logic here. `batch_size = None` was put here by the developer for some purpose, moreover we compare `batch_size` with `None` at  `_get_batch_generator(..)` function